api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -156,6 +156,22 @@ public List<WorkspaceResponse> getPublishedWorkspaces() {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public String getPublishedWorkspacesGroupEmail() {
+    // All users with CT access also have RT access, so we know that any user with access to
```

Comment:
There has been some confusion lately about how "publishing" workspaces work.  It's a bit ... subtle?  hacky?  and I hope this text provides a clear explanation.  Please let me know if we can improve on this.  Thanks!